### PR TITLE
{tools}[GCCcore/8.3.0] UnZip v6.0, Zip v3.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/57a808807479f5f54b9a2558d7e0178f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3104.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/f234e9d24297e08cc4072ff95a8a73fa for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2401.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/96e3614a1dc57c69fd4b9b99a6a0139c for a full test report.

File: easybuild/easyconfigs/z/Zip/Zip-3.0-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+name = 'Zip'
+version = '3.0'
+
+homepage = 'http://www.info-zip.org/Zip.html'
+description = """UnZip is an extraction utility for archives compressed
+in .zip format (also called "zipfiles"). Although highly compatible both
+with PKWARE's PKZIP and PKUNZIP utilities for MS-DOS and with Info-ZIP's
+own Zip program, our primary objectives have been portability and
+non-MSDOS functionality."""
```
Comment (lexming) [inline easybuild/easyconfigs/z/Zip/Zip-3.0-GCCcore-8.3.0.eb]: The description of `Zip` is the one from `Unzip`
```suggestion
description = """Zip is a compression and file packaging/archive utility.
Although highly compatible both with PKWARE's PKZIP and PKUNZIP
utilities for MS-DOS and with Info-ZIP's own UnZip, our primary objectives
have been portability and other-than-MSDOS functionality"""
```

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3219a90e9c1d938729952c905c5f549a for a full test report.

**Comment by lexming** [comment]
Going in, thanks @boegel !

**Pull request merged**